xml_inscriptions/transcribed/NY.NY.NYU.L.48.xml

```diff
@@ -158,38 +165,69 @@
          <change when="2020-10-20" who="Scott J. DiGiulio">Relocated letter dimensions to proper place</change>
          <change when="2021-03-24" who="Elli Mylonas">Change clay.brick to clay.terracotta</change>
          <change when="2023-10-24" who="Advaith Rege">Added images</change>
+         <change when="2024-11-16" who="Sasha Floru">Diplomatic transcription, CIL number, additional commentary</change>
+
       </revisionDesc>
    </teiHeader>
    <!--  *************************************  <facsimile>  *********************************  -->
    <!-- Used to record images of an inscription.   -->
+   
    <facsimile>
       <surface>
          <desc/>
          <graphic url="NY.NY.NYU.L.48.jpg"/>
       </surface>
       <surface>
          <desc></desc>
+
          <graphic url="NY.NY.NYU.L.48 (det.1).jpg"/>
+
       </surface>
    </facsimile>
    <text>
+      
       <body>
+         
+         
+         
          <div type="edition"><!-- transcription goes here -->
-            <ab>
+               
+         <div type="textpart" subtype="diplomatic">
+               
+               <ab>              
+                  <lb n="1"/> C&#183;CVLPI&#183;SVLPIC F SAE              
+                  
+               </ab>
+               
+          </div>          
+              
+         <div type="textpart" subtype="interpretive">  
+               <ab>
                <lb n="1"/>
                <persName>
                   <name type="praenomen" key="Caius">
                 <expan><abbr>C</abbr><ex>aii</ex></expan>
                   </name>
                   <name type="gentilicium" key="Culpius">Culpi</name>
-                  <name type="cognomen" key="Sulpicianus">Sulpic(iani ?)</name></persName>, <expan><abbr>f</abbr><ex>iglinis</ex></expan> <expan><abbr>Sae</abbr><ex>nianis?</ex></expan>.
+                  <name type="cognomen" key="Sulpicianus">Sulpic(iani ?)</name></persName>, <expan><abbr><damage>f</damage></abbr><ex>iglinis</ex></expan> <expan><abbr>Sae</abbr><ex>nianis?</ex></expan>.
```

Comment:
On reflection, I think we should change some of my original markup to look like this: `<name type="cognomen" key="Sulpicianus"><expan><abbr>Sulpic</abbr><ex cert="low">iani</name></persName>, <expan><abbr><damage>f</damage></abbr><ex>iglinis</ex></expan> <expan><abbr>Sae</abbr><ex cert="low">nianis</ex></expan>.`

Reply:
Also re: using `<damage>` here; I thought the f was legible when I marked up and didn't feel the need to note the damage, since it doesn't affect the display at all. If what you're trying to correct to here is to reflect an underdot (i.e. ambiguous reading because of damage), I think what you want there is `<unclear>` instead. Again, I don't think it could be anything else (hence why I didn't mark anything originally); if you want to add the notation that there is some damage to the letter then leave as edited here, and if you're trying to note that the letter is ambiguous (which, again, I'm not convinced that it necessarily is), then you want it to be '<unclear>`